public/app/features/templating/templateSrv.js

```diff
@@ -107,10 +111,15 @@ function (angular, _) {
       });
     };
 
-    this.getAllValue = function(variable) {
+    this.getAllValue = function(variable, format) {
       if (variable.allValue) {
         return variable.allValue;
       }
+
```

Comment:
Line 38 is added exactly for that case.

Without those patches I can't use grafana with influxdb data: "All" value doesn't work because it is escaped, multiple values don't work because of a request size limit. What should I do?